### ISSUE TITLE
MD013: `semantic-line-break` conflict with `mkdocs` PyMdown Blocks

### ISSUE THREAD

**Issue by tahv** [rvben/rumdl#495]
### rumdl version

0.1.40

### Installation method

uv

### Bug description

A spotted a few bugs when MD013 `semantic-line-breaks` is used with `mkdocs` flavor and [PyMDown blocks](https://rumdl.dev/flavors/mkdocs/?h=pymdown#pymdown-blocks).

### Steps to reproduce

`repro.md`:

````markdown
# Repro

<!-- This block is ok -->

/// details | Summary
    type: warning

Content inside the block.

///

<!-- Block with no title trigger diagnostic -->

/// details
    type: warning

Content inside the block.

///

<!-- No newline between options and content trigger diagnostic -->

/// details | Summary
    type: warning
Content inside the block.

///

<!-- No newline between content and end `///` trigger diagnostic -->

/// details | Summary
    type: warning

Content inside the block.
///
````

### Expected behavior

No diagnostics.

### Actual behavior

```console
$ rumdl -V
rumdl 0.1.40

$ rumdl check repro.md
repro.md:14:1: [MD013] Paragraph should use semantic line breaks (1 sentences) [*]    
repro.md:24:1: [MD013] Paragraph should use semantic line breaks (1 sentences) [*]
repro.md:34:1: [MD013] Paragraph should use semantic line breaks (1 sentences) [*]

Issues: Found 3 issues in 1 file (54ms)
Run `rumdl fmt` to automatically fix 3 of the 3 issues
```

### Configuration

`pyproject.toml`:

```toml
[tool.rumdl]
flavor = "mkdocs"
MD013.reflow = true
MD013.reflow-mode = "semantic-line-breaks"
```

### Operating system

Windows

**Comment by rvben** [comment]
Fixed in [v0.1.41](https://github.com/rvben/rumdl/releases/tag/v0.1.41). PyMdown block content is now skipped by MD013's reflow engine. All edge cases from your report are handled. Thanks for reporting @tahv\!
